cypress/e2e/gui/customers.cy.js

```diff
@@ -0,0 +1,187 @@
+const API_URL = Cypress.env('API_URL');
+const CUSTOMERS_URL = `${API_URL}/customers`;
+
+describe('Customers', () => {
+  describe('View', () => {
+    beforeEach(() => {
+      cy.setCookie('cookieConsent', 'accepted');     
+      cy.visit('/');
+    });
+
+    it('It goes back to the customers list when clicking the "Back" button', () => {
```

Comment:
```suggestion
    it('goes back to the customers list when clicking the "Back" button', () => {
```

Recomendo remover a palavra `It`, visto que este é o nome da função (`it`) e é possível usar tal palavra na leitura da descrição do caso de teste.